src/osd/PG.cc

```diff
@@ -4584,12 +4589,12 @@ void PG::_scan_rollback_obs(const vector<ghobject_t> &rollback_obs)
        i != rollback_obs.end();
        ++i) {
     if (i->generation < trimmed_to.version) {
-      osd->clog->error() << "osd." << osd->whoami
-			<< " pg " << info.pgid
-			<< " found obsolete rollback obj "
-			<< *i << " generation < trimmed_to "
-			<< trimmed_to
-			<< "...repaired";
+      dout(10) << __func__ << "osd." << osd->whoami
```

Comment:
@neha-ojha It would be nice if this were logged as an error for replicated pools.

Reply:
@dzafman agreed, will add it in a follow up PR.